src/ext/disable-element.js

```diff
@@ -5,12 +5,14 @@ htmx.defineExtension('disable-element', {
     onEvent: function (name, evt) {
         let elt = evt.detail.elt;
         let target = elt.getAttribute("hx-disable-element");
-        let targetElement = (target == "self") ? elt : document.querySelector(target);
+        let targetElements = (target == "self") ? [ elt ] : document.querySelectorAll(target);
 
-        if (name === "htmx:beforeRequest" && targetElement) {
-            targetElement.disabled = true;
-        } else if (name == "htmx:afterRequest" && targetElement) {
-            targetElement.disabled = false;
+		for (var i = 0; i < targetElements.length; i++) {
```

Comment:
indentation looks a little funky here.  tabs?

Reply:
Yeah, I noticed that in the GitHub editor. The other lines use spaces but I have my IDE set to use tabs instead. In most places the tab size is 4 spaces so it's not noticeable, but for some reason the GitHub editor has like 8.

Reply:
Can you re-indent to match the spaces scheme of the project please?